README.md

```diff
@@ -1,3 +1,10 @@
+The Atomic tool has been deprecated.
+
+The [Podman](https://podman.io) has replaced it with
+most of its functionality implemented. Please open issues and questions
+with the [Podman repository](https://github.com/containers/podman).
+
+
```

Comment:
Has this deprecation been noted in the RHEL release notes somewhere?

Reply:
I have no idea.  I am sure it is mentioned that this package is no longer in RHEL8.